DFC.Digital/DFC.Digital.Services.SendGrid/Services/MergeEmailContentService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using DFC.Digital.Data.Interfaces;
+using DFC.Digital.Data.Model;
+
+namespace DFC.Digital.Services.SendGrid
+{
+    public class MergeEmailContentService : IMergeEmailContent
+    {
+        private const string FirstNameToken = "{firstname}";
+        private const string LastNameToken = "{lastname}";
+        private const string EmailToken = "{email}";
+        private const string ContactOptionToken = "{contactoption}";
+        private const string DobToken = "{dob}";
+        private const string PostCodeToken = "{postcode}";
+        private const string ContactAdvisorQuestionTypeToken = "{contactadviserquestiontype}";
+        private const string MessageToken = "{message}";
+        private const string IsContactableToken = "{iscontactable}";
+        private const string FeedbackQuestionTypeToken = "{feedbackquestiontype}";
+        private const string TermsAndConditionsToken = "{tandc}";
+
+        public string MergeTemplateBodyWithContent(SendEmailRequest sendEmailRequest, string content)
+        {
+            return TokenReplacement(sendEmailRequest, content);
+        }
+
+        public string MergeTemplateBodyWithContentWithHtml(SendEmailRequest sendEmailRequest, string content)
+        {
+            return TokenReplacement(sendEmailRequest, content);
+        }
+
+        private static string TokenReplacement(SendEmailRequest sendEmailRequest, string content)
```

Comment:
As mentioned before `SendEmailRequest` is too close to contact us form, so we need to look at making this dynamic or a dictionary